src/presentation/Console.java

```diff
@@ -169,7 +169,8 @@ public static boolean promptToContinue() {
 			if (matchesAffirmative) {
 				isValid = true;
 			} else if (matchesNegative) {
-				isValid = true;
+				printEmptyLine();
```

Comment:
I don't like adding the `printEmptyLine()` call here to allow for spacing with other things in the app. This function is just supposed to handle asking the user if they'd like to continue and any messages associated with that. If you feel like there should be an additional empty line for good overall formatting, that should go in the file where all of this is actually being called (the application class, in this case).